scenarios/iam_privesc_by_key_rotation/cheat_sheet.md

```diff
@@ -43,10 +40,9 @@ aws iam create-access-key --user-name admin_iam_privesc_by_key_rotation_<cloudgo
 With the "admin" users credentials we can assume the role it has access to...
 
 ```bash
-export AWS_ACCESS_KEY_ID=AKIA....
-export AWS_SECRET_ACCESS_KEY=GQg+9Me8LmB....
+aws configure --profile iam-admin-cloudgoat
```

Comment:
Maybe just have the profile called `admin`, less work to type since it has to be added to every command

Reply:
noted